### PR TITLE
Changes to Ricin acording to feedback

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][balancing]

## About the PR 
This PR adds a potency requirement for ricin rice. This i also changes the emote stage to give you food poisoning instead of custom emotes and deal some small amount of damage.

## Why's this needed? 
Ricin is very easy to make at the moment, and i think a potency requirement while would make it take a bit more time to produce big amounts of ricin, and as it stands, the custom emotes are way too obvious to the point where most poisonings with it will not result in anything, cyanide is easier to get, much faster and has a similar-ish lethal dose so there is really no use for it right now in terms of gameplay. Making it give food poisoning would do a better job at masking the poisoning, while still tipping you off that something isn't right, and just in general making it a more interesting chem.

```changelog
(u)Hans
(+)Makes ricin induce food poisoning instead of custom emotes.
(+)Adds a potency requirement for ricin rice.
```

### CODE REVIEW

**Comment by Eagletanker** [comment]
1400 isn’t a lot. I suggest adding a extra 0 on there

**Comment by colossusqw** [comment]
> 1400 isn’t a lot. I suggest adding a extra 0 on there

well, it's 5 units of ricin, and botany can already produce a lot of ricin, hopefully less with the potency requirement but still. Perhaps 5k for 5 units?

File: code/modules/economy/commodity.dm [hunk 1]
```diff
@@ -797,6 +797,15 @@
 	upperfluc = 100
 	lowerfluc = -100
 
+/datum/commodity/ampoule/ricin
```
[the remark this answers is not in the document]
Comment (colossusqw) [reply code/modules/economy/commodity.dm:800]: 5 units is enough to injure a person severely and will definitely get some of their organs dead or in critical condition, i will change the code as you suggested with the rest, thanks!

**Comment by Tarmunora** [comment]
Eh, don't really like the feel of it being sold from space vendors

**Comment by colossusqw** [comment]
> Eh, don't really like the feel of it being sold from space vendors

If it's concerns with balancing, you can only buy ricin in ampoules, which makes it harder for you to administer them as compared to botany, it costs between 1400 and 2100 for a single 5 unit ampoule, which is about enough to use on one person, and you already see ricin around on poison bottles and (???) pills, so in theory, getting your hands on small amounts of ricin already isn't that hard, + botany will definitely always be by far the most capable of making ricin beyond a few lethal doses, still, if you wish for it not being sold with the vendors, just ask that I'll remove that part of the code.

**Comment by NightmareChamillian** [comment]
Pretty big fan of it being bought in ampoules, since again, non-chem/botany exclusive things are pretty nice and it gives more reason to care about having spacebux beyond "heehoo funny number at end of round"

**Comment by AmazingDragon353** [comment]
> Pretty big fan of it being bought in ampoules, since again, non-chem/botany exclusive things are pretty nice and it gives more reason to care about having spacebux beyond "heehoo funny number at end of round"

Just an fyi that these are credits (what you get from the HoP and budget and whatnot) and not spacebux (what is stored between rounds based on the job you played).

Imo this is a very good change

**Comment by colossusqw** [comment]
!pull upstream

**Comment by colossusqw** [comment]
!merge_upstream

**Pull request merged**